docs/topics/eclipse-installing-plugin.adoc

```diff
@@ -40,6 +48,9 @@ endif::[]
 ifdef::mta[]
 `https://marketplace.eclipse.org/content/migration-toolkit-applications-mta` and click *OK*.
 endif::[]
+. Select the Java version to use.
```

Comment:
@HagayVider1 
Maybe "Select the Java version to use." is unnecessary here, especially since the first bullet (these should be bullets, not numbers) tells to not select anything. I would suggest:
. If you use Java 11, in the JRE list, select *JavaSE 11*.
+
If you use Java 17, leave the *JRE* selection unchanged.